packages/kit/types/private.d.ts

```diff
@@ -184,6 +184,10 @@ export interface PrerenderErrorHandler {
 
 export type PrerenderOnErrorValue = 'fail' | 'continue' | PrerenderErrorHandler;
 
+export type PrerenderOption = boolean | 'auto';
```

Comment:
This should be public API, people might want to do `export let prerender: PrerenderOption = ...` or `myHelperDefiningWhatPrerenderOptionToUse(): PreprenderOption {..}`